capacitor open android return unhandled 'error' event (resolution)

hello everyone, as the title says I was faced with an error during the execution of the command 

> "npx cap open android" 

(under linux) then the console returns an error of the type:

> [info] Opening Android project at /home/yourUserName/yourProjectName/android
> events.js:174
>       throw er; // Unhandled 'error' event
>       ^
> Error: spawn /usr/local/android-studio/bin/studio.sh ENOENT
>     at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
>     at onErrorNT (internal/child_process.js:415:16)
>     at process._tickCallback (internal/process/next_tick.js:63:19)
> Emitted 'error' event at:
>     at Process.ChildProcess._handle.onexit (internal/child_process.js:246:12)
>     at onErrorNT (internal/child_process.js:415:16)
>     at process._tickCallback (internal/process/next_tick.js:63:19)
> [ERROR] An error occurred while running subprocess capacitor.
> 


In fact the problem is quite simple and to solve it just move the android studio folder under the directory "/ user / local /" then relaunch the command ...
PS: the error was displayed because capacitor could not find android studio in the default directory provided to it, that's all.

Here I hope to have help someone good continuation

--- a/cli/src/android/open.ts
+++ b/cli/src/android/open.ts
@@ -38,7 +38,7 @@ export async function openAndroid(config: Config) {
       };
 
       try {
-        await opn(dir, { app: config.linux.androidStudioPath, wait: false });
+        await opn(dir, { app: config.linux.androidStudioPath, wait: true });
       } catch (e) {
         linuxError();
       }

